certbot_dns_loopia.py

```diff
@@ -62,6 +64,7 @@ def _get_loopia_client(self):
             self.credentials.conf("user"),
             self.credentials.conf("password"))
 
+
```

Comment:
There shouldn't be two blank lines between methods. I try to stick to [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines) which states *"Method definitions inside a class are surrounded by a single blank line."*